src/validation_workflow/validation.py

```diff
@@ -38,6 +41,14 @@ def copy_artifact(self, filepath: str, tempdir_path: str) -> bool:
         else:
             raise Exception("Provided path for local artifacts does not exist")
 
+    def check_for_security_plugin(self, work_dir: str, distribution: str) -> bool:
+        list_cmd = "dir" if distribution == "zip" else "ls"
+        (_, plugins_list, _) = execute(f"{list_cmd} plugins", work_dir, True, False)
+        return "opensearch-security" in plugins_list
```

Comment:
Can we just directly check if dir `$OPENSEARCH_HOME/plugins/opensearch-security` exist such as
```
path = '$OPENSEARCH_HOME/plugins/opensearch-security'
 
isExist = os.path.exists(path)
```

This is much easier than running ls/dir and have to specify different OS.